db/20160722_seeds.rb

```diff
@@ -21,7 +21,8 @@ def run
         'Self-starter',
         'Leadership',
         'Perceptive',
-        'Researcher'
+        'Researcher',
```

Comment:
Since this seed is dated to 7/22, it's probably worth making another file that modifies the data. This will help keep a better record of changes (and keep local + staging deployments properly synced).